registry/donor/models.py

```diff
@@ -437,11 +544,42 @@ def refresh_overview(cls):
     JOIN "records"
         ON "records"."id" = "recent_records"."record_id";"""
         )
+
         cls.remove_ignored()
+        cls.refresh_override(commit=False)
         db.session.commit()
 
 
 class Note(db.Model):
     __tablename__ = "notes"
     rodne_cislo = db.Column(db.String(10), primary_key=True)
     note = db.Column(db.Text)
+
+
+class DonorsOverride(db.Model):
+    __tablename__ = "donors_override"
+    rodne_cislo = db.Column(db.String(10), primary_key=True)
+    first_name = db.Column(db.String)
+    last_name = db.Column(db.String)
+    address = db.Column(db.String)
+    city = db.Column(db.String)
+    postal_code = db.Column(db.String(5))
+    kod_pojistovny = db.Column(db.String(3))
+
+    def to_dict(self):
+        result = {}
+        for field in [
```

Comment:
Bylo by super to udělat tak, aby se seznam vstupů nemusel udržovat na dvou místech - DonorsOverride model a DonorsOverrideForm.